packages/terra-site/webpack.prod.config.js

```diff
@@ -16,9 +17,13 @@ config.plugins.push(new webpack.DefinePlugin({
 // Clean build before running
 config.plugins.push(new CleanPlugin('build', { exclude: ['stats.json'] }));
 
+// Minify css and js
+config.plugins.push(new webpack.LoaderOptionsPlugin({ minimize: true }));
+
```

Comment:
Full tree-shaking support with webpack 2 is a bit involved, and there are some issues we need to consider around using UglifyJsPlugin vs babili plugin. With using the UglifyJsPlugin, we could do partial tree shaking, (unused functions are removed, but not unused classes). This requires a config change in the [babelrc file](https://gist.github.com/blacksonic/e7799a87dda003c5f4da4f6eae85336c/raw/32a618c4d8af5602729bb616b33b984d8413c5d4/.babelrc) and the [webpack config](https://gist.github.com/blacksonic/70078c33e3dd3a25615c9236932c9fb2/raw/fe195b0845d75ff7ca333a08cd134c42486ff7f0/webpack.prod.config.js) to work. 

I'd like to explore using the babili plugin instead of UglifyJsPlugin so we can also remove unused classes trim the size even more.

This is a really good write up on webpack 2 tree-shaking and whats all involved to get full tree-shaking support: https://blog.craftlab.hu/how-to-do-proper-tree-shaking-in-webpack-2-e27852af8b21

More info on babili: https://babeljs.io/blog/2016/08/30/babili

I'm also ok if we want to merge this down as is now and explore tree-shaking functionality in a separate PR or handle it in this PR.